### PR TITLE
Add Implemantation of Twitter Trigger

### PR DESCRIPTION
**Pull request by kapilraajP**
```
public type TweetsService service object {
    remote function onTweetsCreate(TweetEvent event) returns error?;
    remote function onTweetsDelete(TweetDeleteEvent event) returns error?;
    remote function onFollow(FollowEvent event) returns error?;
    remote function onFavorite(FavoriteEvent event) returns error?;
};
```

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
  Ballerina Version: 2201.0.4
  Operating System: Ubuntu 20.04
  Java SDK: 11

### CODE REVIEW

File: asyncapi/twitter/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Cost/Freemium" , "Vendor/Twitter"]
```
Comment (RolandHewage) [inline asyncapi/twitter/Ballerina.toml:3]: Let's add the missing keyword "Trigger" here

File: asyncapi/twitter/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+org = "ballerinax"
+name = "trigger.twitter"
+repository = "https://github.com/ballerina-platform/asyncapi-triggers"
+version = "0.1.0"
```
Comment (RolandHewage) [inline asyncapi/twitter/Ballerina.toml:7]: Let's go with a major release 1.0.0

File: asyncapi/twitter/Module.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+## Overview
+This is a trigger for [Twitter](https://developer.twitter.com/en/docs/twitter-api/enterprise/account-activity-api/api-reference) that is capable of listening to the following events that occur in Twitter. 
+- Tweeted
+- Deleteed Tweet
```
Comment (SkNuwanTissera) [inline asyncapi/twitter/Module.md:4]:
```suggestion
- Deleted Tweet
```

File: asyncapi/twitter/data_types.bal [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+    string? created_timestamp?;
+    User? target?;
+    User? 'source?;
+};
+
+# Information about favorite event
+#
+# + for_user_id - User ID  
+# + favorite_events - Description about event when someone make tweet favorited
+public type FavoriteEvent record {
```
Comment (SkNuwanTissera) [inline asyncapi/twitter/data_types.bal:89]:
```suggestion
public type FavouriteEvent record {
```

File: asyncapi/twitter/Ballerina.toml (hunk 1 above)
Comment (kapilraajP) [reply asyncapi/twitter/Ballerina.toml]: Added that Keyword

File: asyncapi/twitter/Ballerina.toml (hunk 2 above)
Comment (kapilraajP) [reply asyncapi/twitter/Ballerina.toml]: Ack. Version is updated as major release

File: asyncapi/twitter/Module.md (hunk 3 above)
Comment (kapilraajP) [reply asyncapi/twitter/Module.md]: Typo is corrected

File: asyncapi/twitter/data_types.bal (hunk 4 above)
Comment (kapilraajP) [reply asyncapi/twitter/data_types.bal:89]: Since events received from Twitter are with the name `favorite_events` used the same name. WDYT?

**Comment by Drifftr** [comment]
Closing since this implementation is outdated.

**Pull request closed without merging**